### DISCUSSION TITLE
希望可以支持MySQL/MariaDB [数据库]

### DISCUSSION THREAD

**Discussion by xiangxiangxiong9** [Mrs4s/go-cqhttp#709] [Ideas]
刚刚遇到了 [FATAL]: 打开数据库失败, 如果频繁遇到此问题请清理 data/leveldb 文件夹或关闭数据库功能。 ...
刚好本地就有个MariaDB...希望可以将数据存放到数据库里...

**Comment by Mrs4s** [comment]
增加多个数据库支持会导致编译出来的二进制体积增加不少, 今后考虑吧模块化编译做出来后再做这个

**Reply by xiangxiangxiong9** [reply]
莫名奇妙的因为这玩意崩溃了 85 次 ...也不知道咋回事...

**Reply by Mrs4s** [reply]
上错误日志看看

**Reply by xiangxiangxiong9** [reply]
我想让那玩意出来它不出来…
不想让它出来偏出来…
暂时没法复现…[玄学问题…]
